src/analysis/seminorm.lean

```diff
@@ -322,20 +306,20 @@ variables [semi_normed_comm_ring 𝕜] [add_comm_group E] [add_comm_group F] [mo
 
 lemma comp_smul (p : seminorm 𝕜 F) (f : E →ₗ[𝕜] F) (c : 𝕜) :
   p.comp (c • f) = ∥c∥₊ • p.comp f :=
-ext $ λ _, by rw [comp_apply, smul_apply, linear_map.smul_apply, p.smul, nnreal.smul_def,
+ext $ λ _, by rw [comp_apply, smul_apply, linear_map.smul_apply, map_smul_eq_mul, nnreal.smul_def,
   coe_nnnorm, smul_eq_mul, comp_apply]
 
 lemma comp_smul_apply (p : seminorm 𝕜 F) (f : E →ₗ[𝕜] F) (c : 𝕜) (x : E) :
-  p.comp (c • f) x = ∥c∥ * p (f x) := p.smul _ _
+  p.comp (c • f) x = ∥c∥ * p (f x) := map_smul_eq_mul p _ _
 
 end semi_normed_comm_ring
 
 section normed_field
-variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E]
+variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] {p q : seminorm 𝕜 E} {x : E}
 
-private lemma bdd_below_range_add (x : E) (p q : seminorm 𝕜 E) :
-  bdd_below (range (λ (u : E), p u + q (x - u))) :=
-by { use 0, rintro _ ⟨x, rfl⟩, exact add_nonneg (p.nonneg _) (q.nonneg _) }
+/-- Auxiliary lemma to show that the infimum of seminorms is well-defined. -/
+lemma bdd_below_range_add : bdd_below (range $ λ u, p u + q (x - u)) :=
```

Comment:
I'm not convinced that the parameters here should become implicit, although the invocations don't need them, they also look pretty hard to infer in general. But I don't know enough about these corners of the library to see whether the lemma will be used in more complicated situations.

Reply:
My expectation is that this will be used < 5 times and always in the situation of proving that the infimum of seminorms is well-behaved, in which case all the information is already in the goal and implicits will be inferred painlessly.